src/common/utils/api/api.ts

```diff
@@ -122,14 +150,20 @@ export default {
   getResource: (id: string): Promise<Resource> =>
     apiGet<Resource>({ path: `${resourceBasePath}/${id}` }),
 
-  getDatePeriod: (resourceId: string): Promise<DatePeriod[]> =>
```

Comment:
I think the resource id can also be in format "tprek:8100" 🤔

Reply:
Or does it mean internal id?

Reply:
I think it is the internal id here. Atleast the way it is applied in ResourcePage:

```
useEffect((): void => {
      if (resource) {
        api
          .getDatePeriod(resource.id)
          .then((ds: DatePeriod[]) => {
            setDatePeriods(ds);
          })
          .catch((e: Error) => {
            setError(e);
          });
      }
    }, [resource]);
```

And if we look at the Resource returned by api here: http://hauki-test.oc.hel.ninja/v1/resource/tprek:8215/ It shows:

![image](https://user-images.githubusercontent.com/2777633/99549975-e4907a00-29c2-11eb-8246-baaf654fee6a.png)

It seems we can fetch a resource by applying the 'tprek:8215' format in the url, but then the resource that we get has an id with number format. I changed the format to number in the Resource type definition also.